BREAKING.md

```diff
@@ -24,6 +24,21 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 The `"batchEnd"` listener in `ISummarizerRuntime` has been deprecated and will be removed in a future release. Please remove all usage and implementations of `ISummarizerRuntime.on("batchEnd", ...)` and `ISummarizerRuntime.removeListener("batchEnd", ...)`.
 If these methods are needed, please refer to the `IContainerRuntimeBase` interface.
 
+### Deprecated events and event parameters on IContainer and IDeltaManager
+
+The following legacy events and event parameters have been marked as deprecated:
+
+- IContainerEvents
+    - "contextChanged": Event deprecated in its entirety. No replacement API recommended.
+    - "dirty": Event parameter "dirty" deprecated. The parameter is unneeded, as the event itself signals the current "dirty" state (true).
+    - "saved": Event parameter "dirty" deprecated. The parameter is unneeded, as the event itself signals the current "dirty" state (false).
+- IDeltaManagerEvents
+    - "prepareSend": Event deprecated in its entirety. No replacement API recommended.
+    - "submitOp": Event deprecated in its entirety. No replacement API recommended.
+    - "allSentOpsAckd": Event deprecated in its entirety. Use "saved" event instead.
```

Comment:
I would either state this is 1:1 substitution or describe (any) differences here,

Reply:
Can you elaborate? I'm not sure what you mean.

Reply:
If `allSentOpsAckd` is deprecated, and `saved` should be used instead - are there any differences in behavior that consumer should know about?

Reply:
Ah, I see. I didn't realize the comment was about the single event. Let me double check and update as needed.